package.json

```diff
@@ -58,7 +59,7 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.27.1",
     "husky": "^7.0.4",
-    "jsdoc": "^3.6.7",
+    "jsdoc": "^4.0.2",
```

Comment:
Why didn't you revert this as Bennett requested?  Please do that now, then request a new review.